src/pages/StoryForm/StoryForm1.css

```
@@ -60,8 +60,9 @@
 
   color: var(--color-green-main);
 
-  font-size: 26px;
-  font-weight: 600;
+  font-family: Warhaven, sans-serif;
```

Stylelint (17.14.0)

[error] Expected no empty line before declaration (declaration-empty-line-before)

(declaration-empty-line-before)